src/components/Searchbar.svelte

```diff
@@ -18,9 +70,15 @@
 </style>
 
 <!-- svelte-ignore a11y-autofocus -->
-<input
-  maxlength="70"
-  type="text"
-  placeholder="Search your gitmoji..."
-  autofocus
-  on:input />
+<div class="input">
+  <input
+    maxlength="70"
```

Comment:
Why providing a maxlength ?

Reply:
A longer filter value make no sense here. So just avoid the user to stay presses on a random key 😂